File: app/src/main/java/com/wojdor/hearthcards/application/classpager/ClassPagerViewModel.java
package com.wojdor.hearthcards.application.classpager;

import android.app.Application;
import android.arch.lifecycle.LiveData;
import android.arch.lifecycle.MutableLiveData;
import android.support.annotation.NonNull;

import com.wojdor.hearthcards.application.base.BaseAndroidViewModel;
import com.wojdor.hearthcards.domain.VersionInfo;

public class ClassPagerViewModel extends BaseAndroidViewModel {

    public ClassPagerViewModel(@NonNull Application application) {
        super(application);
    }

    public LiveData<VersionInfo> getLocalVersionInfo() {
        MutableLiveData<VersionInfo> data = new MutableLiveData<>();
        data.setValue(userSession.getVersionInfo());
        return data;
    }
}

File: app/src/main/java/com/wojdor/hearthstonecards/application/classpager/ClassPagerViewModel.kt
package com.wojdor.hearthstonecards.application.classpager

import android.app.Application
import android.arch.lifecycle.LiveData
import com.wojdor.hearthstonecards.application.base.BaseAndroidViewModel

class ClassPagerViewModel(application: Application) : BaseAndroidViewModel(application) {

    val classesWhichHaveCards: LiveData<List<String>>
        get() = repository.classesWhichHaveCards
}
